### PR TITLE
ContextMenu: Update target area sizes for JS Frameworks

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

File: apps/demos/Demos/ContextMenu/Scrolling/Angular/app/app.component.css [hunk 1]
```diff
@@ -1,10 +1,21 @@
-.target-area {
+::ng-deep .demo-container {
+  position: absolute;
+  top: 8px;
+  left: 8px;
+  right: 8px;
+  bottom: 8px;
+  height: 100%;
+}
```
Comment (ksercs) [inline apps/demos/Demos/ContextMenu/Scrolling/Angular/app/app.component.css:13]: Why do we need this additional styles expect height=100%?

**Comment by jdvictoria** [comment]
[ContextMenu demo: update target area sizes](https://trello.com/c/G1zAlpL5/8822-contextmenu-demo-update-target-area-sizes)

**Comment by jdvictoria** [comment]
[rework export custom markup demo - replace strings using to elements](https://trello.com/c/hXcC8zFr/8672-rework-export-custom-markup-demo-replace-strings-using-to-elements)

**Review by iBat — DISMISSED** [review]

File: apps/demos/Demos/ContextMenu/Scrolling/Angular/index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
     <link rel="stylesheet" type="text/css" href="../../../../../../node_modules/devextreme-dist/css/dx.light.css" />
+    <link rel="stylesheet" type="text/css" href="./app/app.component.css" />
```
Comment (ksercs) [inline apps/demos/Demos/ContextMenu/Scrolling/Angular/index.html]: Why did u added this? Css is already bounded using `styleUrls: ['app/app.component.css'],` configuration in `app.components.ts`

**Review by ksercs — DISMISSED** [review]

File: apps/demos/Demos/ContextMenu/Scrolling/Angular/index.html (hunk 2 above)
Comment (jdvictoria) [reply apps/demos/Demos/ContextMenu/Scrolling/Angular/index.html]: Resolved

File: apps/demos/Demos/ContextMenu/Scrolling/Angular/app/app.component.css [hunk 3]
```diff
@@ -1,10 +1,16 @@
+::ng-deep body {
```
Comment (ksercs) [inline apps/demos/Demos/ContextMenu/Scrolling/Angular/app/app.component.css:7]: this css does not work in our Widget Gallery

File: apps/demos/Demos/ContextMenu/Scrolling/React/styles.css [hunk 4]
```diff
@@ -1,10 +1,30 @@
...
+
+body {
+  margin: 0;
+}
+
+#app {
+  height: 100%;
+}
+
+.demo-container {
```
Comment (ksercs) [inline apps/demos/Demos/ContextMenu/Scrolling/React/styles.css:12]: cannot we union this rules for app and demo-container?

File: apps/demos/Demos/ContextMenu/Scrolling/Vue/App.vue [hunk 5]
```diff
@@ -23,14 +21,28 @@ function itemClick({ itemData }: DxContextMenuTypes.ItemClickEvent) {
 }
 </script>
 <style>
+html,
+body,
+#app {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+}
```
Comment (ksercs) [inline apps/demos/Demos/ContextMenu/Scrolling/Vue/App.vue:32]: hm, why do u do it in a different way comparing with other frameworks?

**Review by iBat — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**